### ISSUE TITLE
Pylint reports false errors with RPi.GPIO

### ISSUE THREAD

**Issue by afeser** [PyCQA/pylint#7567]
### Bug description

Existing attributes in `RPi.GPIO` are not recognized in Pylint and I always get "no-member" error.

The script is(pylint_test.py):
```python3
"""some docstring"""
from RPi import GPIO

print(GPIO.LOW)
```

```bash
python3 -m pylint pylint_test.py
```

I tried with 2 pylint versions, the output is identical.

### Command used

```shell
# Tried 2 commands:
pylint3 pylint_test.py

# after installing `pip3 install pylint`
python3 -m pylint pylint_test.py
```

### Pylint output

```shell
************* Module pylint_test
pylint_test.py:4:6: E1101: Module 'RPi.GPIO' has no 'LOW' member (no-member)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior

No errors, the code works without any problems and `LOW` attribute exists.

### Pylint version

```shell
# pylint3 --version
pylint3 2.2.2
astroid 2.1.0
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0]

# python3 -m pylint --version
pylint 2.15.3
astroid 2.12.10
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0]
```

### OS / Environment

Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster

Raspberry Pi 4B

Python 3.7.3 venv

### Additional dependencies

RPi.GPIO==0.7.1

**Comment by bchardin** [comment]
Does the problem still exist with the `--extension-pkg-allow-list=RPi` option?  
The linting has to be performed on a Raspberry Pi for the RPi module to be loaded correctly.

**Comment by Pierre-Sassoulas** [comment]
pylint is working as intended if ``RPi`` is not available in the environment you're launching pylint in. You should probably disable the no-member if launching pylint on the Raspberry Pi is not feasible.

**Comment by afeser** [comment]
Thanks for the replies

@Pierre-Sassoulas 
I was running in the Pi

@bchardin 
> Does the problem still exist with the `--extension-pkg-allow-list=RPi` option? The linting has to be performed on a Raspberry Pi for the RPi module to be loaded correctly.

Yes it works with the flag. 
So, it seems this is not a bug, and I am closing the issue, sorry for the confusion..
